ome/omero-iviewer
# Mobile layout

Support iviewer use on mobile phones by switching the layout on a small screen to a "column" instead of a "row":
(layout should be unchanged on regular sized screens)

To test:
You can either test the layout on a small screen using a mobile phone or you can emulate one using the browser dev tools:
 - With a small screen, check that the layout is 'vertical' as shown, and the top and bottom panels can be collapsed & expanded
 - Thumbnails should still be scrollable etc. Viewer, rendering controls etc should be working.
 - On a big screen, everything should be mostly unchanged. Right panel drag-resize should still be working. Left panel resize has been removed (wasn't really doing much before).

<img width="383" alt="Screenshot 2024-10-24 at 23 34 43" src="https://github.com/user-attachments/assets/2155c77c-6370-4ba1-bbf5-2a4300fc5b3d">
